firebase_admin/_auth_client.py

```diff
@@ -0,0 +1,625 @@
+                ``auth.DELETE_ATTRIBUTE``.
+            password: The user's raw, unhashed password. (optional).
+            disabled: A boolean indicating whether or not the user account is disabled (optional).
+            custom_claims: A dictionary or a JSON string contining the custom claims to be set on
+                the user account (optional). To remove all custom claims, pass
+                ``auth.DELETE_ATTRIBUTE``.
+            valid_since: An integer signifying the seconds since the epoch (optional). This field
+                is set by ``revoke_refresh_tokens`` and it is discouraged to set this field
+                directly.
+
+        Returns:
+            UserRecord: An updated UserRecord instance for the user.
+
+        Raises:
+            ValueError: If the specified user ID or properties are invalid.
+            FirebaseError: If an error occurs while updating the user account.
+        """
+        self._user_manager.update_user(uid, **kwargs)
+        return self.get_user(uid=uid)
+
+    def set_custom_user_claims(self, uid, custom_claims):
+        """Sets additional claims on an existing user account.
+
+        Custom claims set via this function can be used to define user roles and privilege levels.
+        These claims propagate to all the devices where the user is already signed in (after token
+        expiration or when token refresh is forced), and next time the user signs in. The claims
+        can be accessed via the user's ID token JWT. If a reserved OIDC claim is specified (sub,
+        iat, iss, etc), an error is thrown. Claims payload must also not be larger then 1000
+        characters when serialized into a JSON string.
+
+        Args:
+            uid: A user ID string.
+            custom_claims: A dictionary or a JSON string of custom claims. Pass None to unset any
+                claims set previously.
+
+        Raises:
+            ValueError: If the specified user ID or the custom claims are invalid.
+            FirebaseError: If an error occurs while updating the user account.
+        """
+        if custom_claims is None:
+            custom_claims = _user_mgt.DELETE_ATTRIBUTE
+        self._user_manager.update_user(uid, custom_claims=custom_claims)
+
+    def delete_user(self, uid):
+        """Deletes the user identified by the specified user ID.
+
+        Args:
+            uid: A user ID string.
+
+        Raises:
+            ValueError: If the user ID is None, empty or malformed.
+            FirebaseError: If an error occurs while deleting the user account.
+        """
+        self._user_manager.delete_user(uid)
+
+    def import_users(self, users, hash_alg=None):
+        """Imports the specified list of users into Firebase Auth.
+
+        At most 1000 users can be imported at a time. This operation is optimized for bulk imports
+        and will ignore checks on identifier uniqueness which could result in duplications. The
```

Comment:
Should this be "ignores checks on identifier uniqueness, which could result in duplications." or

"checks on identifier uniqueness that could result in duplications."

If there are some check that don't result in dupes, it should be the second one I think.  If all such checks result in dupes, the first.

Reply:
It's the first. Changed as suggested.